R/drive_cp.R

```diff
@@ -96,13 +96,13 @@ drive_cp <- function(file,
   # load (path, name) into params
   if (!is.null(path)) {
     path <- as_parent(path)
-    params[["parents"]] <- list(path$id)
+    params[["parents"]] <- list(as.character(path$id))
```

Comment:
There's lots of this explicit `as.character()` business now, to appease jsonlite.

```{r eval = FALSE}
ids
#> <drive_id[12]>
#>  [1] Jan Feb Mar Apr May Jun Jul Aug Sep Oct Nov Dec
jsonlite::toJSON(ids)
#> Error: No method asJSON S3 class: vctrs_vctr
```

Reply:
Can you provide a method for `asJSON`? Or maybe this is just fixed by adding the `character` class? Either way, we should talk to Jeroen and see if there's a jsonlite tweak to be made.